pkg/console/util.go

```diff
@@ -512,6 +515,25 @@ func doInstall(g *gocui.Gui, hvstConfig *config.HarvesterConfig, webhooks Render
 		env = append(env, fmt.Sprintf("HARVESTER_ADDITIONAL_KERNEL_ARGUMENTS=%s", hvstConfig.OS.AdditionalKernelArguments))
 	}
 
+	// when WipeAllDisks is enabled then find all non installation disks with COS_ prefixed labels
+	// and add them to a list for wiping
+	if hvstConfig.Install.WipeAllDisks {
+		diskOpts, err := getWipeDisksOptions(hvstConfig)
+		if err != nil {
+			return err
+		}
+		for _, opt := range diskOpts {
+			hvstConfig.Install.WipeDisksList = append(hvstConfig.Install.WipeDisksList, opt.Value)
+		}
+	}
+
+	// prepare to wipe disks
+	for _, disk := range hvstConfig.Install.WipeDisksList {
```

Comment:
Will it be faster to perform each iteration with a goroutine?

Reply:
the actual wipe barely takes a second which is a not much in the overall time taken during the installation process